<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OOSpec V0.0  </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyPlot" name="plotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startCollectButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Collect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="focusButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Focus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopFocusButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Focus</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="SpecTab">
     <attribute name="title">
      <string>Spectrometer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Integration Time</string>
      </property>
      <widget class="QLineEdit" name="integTimeLE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>msec</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lessLessITButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lessITButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="moreITButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="moreMoreITButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Scans To Average</string>
      </property>
      <widget class="QSpinBox" name="scansToAvgSB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Output File </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FitTab">
     <attribute name="title">
      <string>Fit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Rt. Peak</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>13</y>
         <width>171</width>
         <height>76</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Amplitude :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="amp0LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Peak X Loc :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="peak0LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Width :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="width0LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>100</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Lt. Peak</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>20</y>
         <width>171</width>
         <height>76</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Amplitude :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="amp1LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Peak X Loc :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="peak1LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Width :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="width1LE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Fit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>200</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Linear Base</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>20</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Slope :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="slopeLE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Intercept :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="intcptLE">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>621</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Locations</string>
    </property>
    <widget class="QLineEdit" name="xposLE">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yposLE">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>VLine Position :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vlineLE">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="testdataButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Testdata</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPlot</class>
   <extends>QWidget</extends>
   <header>myplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
